python/ray/autoscaler/_private/node_launcher.py

```diff
@@ -74,6 +83,7 @@ def run(self):
                 self._launch_node(config, count, node_type)
             except Exception:
                 self.prom_metrics.node_launch_exceptions.inc()
+                self.prom_metrics.failed_create_nodes.inc(count)
```

Comment:
Are these metrics the same?

Reply:
Sort of, failed_create_nodes increments by the number of nodes in the batch whenever a launch fails (number of nodes we tried to launch, but failed due to exception), while node_launch_exceptions always increments by 1 (number of exceptions). So if we only launch 1 node at a time they would be the same

Reply:
OOps, totally missed this!